### PR TITLE
#1211: Add warning for the time of StAX use (CVE-2015-3192)

### PR DESCRIPTION
**Pull request by kazuki43zoo**
Please review #1211 .

### CODE REVIEW

File: source_en/ArchitectureInDetail/Ajax.rst [hunk 1]
```diff
@@ -34,6 +34,13 @@ How to use
 
 Application settings
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Application settings for Ajax are explained below.
+
+.. warning:: **DoS attack measures at the time of StAX(Streaming API for XML) use**
+
+    If the StAX is used to parse the XML format data, DoS attack measures is required.
```
Comment (making) [inline source_en/ArchitectureInDetail/Ajax.rst]: DoS attack measures is required. -> protect DoS attack

File: source_en/ArchitectureInDetail/REST.rst [hunk 2]
```diff
@@ -1637,6 +1637,12 @@ Application settings
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Application settings for RESTful Web Service are explained below.
 
+.. warning:: **DoS attack measures at the time of StAX(Streaming API for XML) use**
+
+    If the StAX is used to parse the XML format data, DoS attack measures is required.
```
Comment (making) [inline source_en/ArchitectureInDetail/REST.rst]: DoS attack measures is required. -> protect DoS attack

**Comment by kazuki43zoo** [comment]
I've reflected review comments.
Please review again.

**Comment by making** [comment]
please backport

**Pull request merged**